docs/prepping-local-system-runbook-windows.md

```diff
@@ -255,6 +255,10 @@ For the `platform/installer` image, the :tag represents the version of Pega you
 
 For `platform/pega` and `platform/search` images, Pega also offers an image with a version tag appended with a datestamp using the pattern `pegaVersion-YYYYMMDD` to indicate the version and the date that Pega built the image. For example, if you pull the `platform/pega` with a tag, `pega:8.5.1-20201026`, the tag indicates that Pega built this 8.5.1 image on 26 October 2020. Using the version tag without the datestamp will always point to the most recently built image for that version.
 
+The Docker image `platform/clustering-service` is used with the Hazelcast subchart under [Pega charts](charts/pega/charts). Using the Clustering Service docker image, Pega platform and Hazelcast can be deployed in a Client Server arrangement. **Pega supports Client Server deployment with Hazelcast from Pega "8.6".**
```

Comment:
For Pega Platform 8.6 and later, Pega supports a new client-server deployment model using a cluster of nodes running Hazelcast as server, 
to which the Pega nodes connect as client. This new client-server deployment model introduces improved stability for Pega Platform 
deployments that use a large number of nodes, since it introduces independent scalability for both servers and clients in Pega Platform.
Because of the inherent stability and scalability of this model, Pega recommends that all clients deploying Pega platform 8.6 and later adopt 
this client-server model using Hazelcast.
To deploy Pega platform and Hazelcast using this client-server deployment model, you must download and push the Clustering Service Docker image, 
`platform/clustering-service` to your Docker repository and use with the Hazelcast subchart under [Pega charts](charts/pega/charts).